### DISCUSSION TITLE
Is there any timeline for next release?

### DISCUSSION THREAD

**Discussion by mksha** [argoproj-labs/applicationset#225] [Q&A] [answered]
When we will be releasing the next release for ApplicationSet controller? @jgwest  @OmerKahani

**Comment by jgwest** [comment, marked as answer]
Hi @mksha, we will tend to release at least as often as Argo CD, whose next release can be tracked [in Argo CD v2.1 milestone](https://github.com/argoproj/argo-cd/milestone/22). We do have the option of releasing sooner, based on the quantity of contributions and if we feel the release is sufficiently stable.

In the mean time, folks can build and host their own images like so:
```bash
# Clone the ApplicationSet repo, and checkout the desired commit.

# Build and push the image
IMAGE_NAMESPACE='(your-quay.io or docker hub-username)' CONTAINER_REGISTRY='(quay.io or dockerio.io)' IMAGE_TAG='(version)' image-push
# Example: IMAGE_NAMESPACE=jgwest-redhat CONTAINER_REGISTRY=quay.io IMAGE_TAG=latest image-push

kubectl apply -f manifest/install.yaml
```

**Reply by mksha** [reply]
thanks for info, and will try the custom build also :).

**Reply by mksha** [reply]
It worked like a charm, thanks a lot

**Reply by mksha** [reply]
@jgwest seems like we just released the argocd version 2.2.0 , can you also release the applicationset next release pls? As you mentioned last time we release it with ArgoCD. So seems like it should also be released.

**Reply by mksha** [reply]
@alexmt

**Reply by jgwest** [reply]
Hi @mksha 2.0.2 was released, which is only a maintenance release, see the milestone link above for tracking the next proper Argo CD release.

**Reply by OmerKahani** [reply]
@jgwest maybe we should create a development image from main on every commit? we had many bug fixes and new features since the last version - and a lot of people are waiting for them.

**Reply by jgwest** [reply]
@OmerKahani 100% agree that would be useful :+1:, I'm actually working on it, this is the first step: https://github.com/argoproj-labs/applicationset/pull/242
